packages/protocol/package.json

```diff
@@ -6,61 +6,61 @@
-    "download-artifacts": "./scripts/bash/download_artifacts.sh",
-    "init-network": "./scripts/bash/init_network.sh",
-    "upload-artifacts": "./scripts/bash/upload_artifacts.sh",
-    "upgrade": "./scripts/bash/upgrade.sh",
-    "revoke": "./scripts/bash/revoke.sh",
+    "devchain": "yarn ts-node scripts/devchain.ts",
+    "devchain:reset": "yarn devchain generate-tar .tmp/devchain.tar.gz --upto 29",
+    "ganache-dev": "./scripts/bash/ganache.sh",
+    "ganache-devchain": "./scripts/bash/ganache_devchain.sh",
+    "generate-stabletoken-files": "yarn ts-node ./scripts/generate-stabletoken-files.ts",
     "govern": "./scripts/bash/govern.sh",
-    "console": "./scripts/bash/console.sh",
-    "check-versions": "./scripts/bash/check-versions.sh",
-    "check-opcodes": "yarn ts-node scripts/check-opcodes.ts",
+    "init-network": "./scripts/bash/init_network.sh",
+    "lint": "yarn run lint:ts && yarn run lint:sol",
+    "lint:sol": "solhint --version && solhint './contracts/**/*.sol' && solhint './contracts-0.8/**/*.sol'",
+    "lint:ts": "yarn run --top-level eslint .",
     "make-release": "./scripts/bash/make-release.sh",
-    "verify-deployed": "./scripts/bash/verify-deployed.sh",
-    "verify-release": "./scripts/bash/verify-release.sh",
+    "migrate": "./scripts/bash/migrate.sh",
+    "prepare-contracts-and-abis-publishing": "yarn ts-node --preferTsExts ./scripts/prepare-contracts-and-abis-publishing.ts",
+    "prepare-devchain-anvil-publishing": "yarn ts-node --preferTsExts ./scripts/change-anvil-devchain-package-version.ts",
+    "revoke": "./scripts/bash/revoke.sh",
+    "set-block-gas-limit": "./scripts/bash/set_block_gas_limit.sh",
     "size:onchain": "./scripts/bash/get_smart_contract_size_from_onchain_address.sh",
     "size:artifacts": "./scripts/bash/get_smart_contract_size_from_build_artifacts.sh",
-    "ganache-dev": "./scripts/bash/ganache.sh",
-    "ganache-devchain": "./scripts/bash/ganache_devchain.sh",
+    "sourcify-publish": "yarn ts-node ./scripts/sourcify-publish.ts",
+    "submodules:pull": "git submodule update --init --recursive",
+    "submodules:delete": "rm -rf $(git submodule | awk '{ print $2 }')",
+    "tags:compare": "./scripts/bash/compare-git-tags.sh",
+    "tags:view": "git for-each-ref 'refs/tags/core-contracts.*' --sort=-committerdate --format='%(color:magenta)%(committerdate:short) %(color:blue)%(tree) %(color:green)github.com/celo-org/celo-monorepo/releases/tag/%(color:yellow)%(refname:short)'",
+    "test": "rm test/**/*.js ; node runTests.js",
+    "test:coverage": "yarn run test --coverage",
+    "test:gas": "yarn run test --gas",
+    "test:generate-old-devchain-and-build": "./scripts/bash/generate-old-devchain-and-build.sh",
+    "test:quicktest": "./scripts/bash/quicktest.sh",
+    "test:release-snapshots": "./scripts/bash/release-snapshots.sh",
+    "test:scripts": "yarn ts-node scripts/run-scripts-tests.ts --testPathPattern=scripts/",
```

Comment:
All those tests should be verified if we use them at all and I would appreciate adding `forge test` as a default ones for local development

Reply:
I introduced with `yarn test` which works with Foundry and I renamed old test as `yarn test:truffle`.
Potentially good idea would be to revisit later those and delete as much as we can after removing dependencies on Truffle and devchain. Resolving for now 😄